src/si/couplers.stanza

```diff
@@ -95,4 +95,36 @@ public pcb-module symmetric-pull-downs (comp:Instantiable -- margin:Double = 0.0
   val y-off = (y(cyard) + margin) / 2.0
 
   place(R[0]) at loc(0.0, y-off, 180.0) on Top
-  place(R[1]) at loc(0.0, (- y-off)) on Top
\ No newline at end of file
+  place(R[1]) at loc(0.0, (- y-off)) on Top
+
+doc: \<DOC>
+Construct Aligned Symmetric Shunt Components
+
+This function constructs a shunt module where two 2-pin components
+are instantiated with p[1] of both components adjacent to each other.
+The far pin (p[2]) on both components is connected to a `COMMON` terminal 
+which could be either a ground or a power supply net.
+
+@param comp 2-pin component like a resistors/capacitor with `p[1]` and `p[2]`
+@param margin Additional Y spacing distance between the two instantiated components.
+
+<DOC>
+public pcb-module symmetric-shunt (comp:Instantiable -- margin:Double = 0.0) :
```

Comment:
`symmetric-pull-downs` and `symmetric-shunt` are basically the same - you just changed `GND` to `COMMON` right ? 

If that is the case - then lets just rename `symmetric-pull-downs` and to `symmetric-shunt`. I like that name better and it is more versatile. 

The only place that `symmetric-pull-downs` is used is USB C converter and it isn't released - so I'll just rename there when this merges.

Reply:
I didn't want to change a function name without asking 😄